prediction_market_agent_tooling/agent/development_tools/test_all_prophet_agents.py

```
@@ -0,0 +1,190 @@
+
+def execute_prophet_predict(
+    agent: Agent, include_reasoning: bool = False
+) -> partial[PredictionProphet]:
+    return partial(make_prediction, include_reasoning=include_reasoning)
+
+
+def test_all_models(
+    dataset_path: str,
+    max_trades_to_test_on: int = 3000,
+    delay_between_trades: float = 2.0,
+    max_bet_amount_min: USD = USD(1),
```

Ruff (0.11.9)

Do not perform function call `USD` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)